bindings/python/dbr_module/errorcodes.py

```diff
@@ -103,7 +105,9 @@
   DBR_ERR_INVALIDOP, // invalid operation
   DBR_ERR_BE_POST, // posting request to back-end failed
   DBR_ERR_BE_GENERAL, // Unspecified back-end error
-  DBR_ERR_MAXERROR
+  DBR_ERR_ITERATOR,
+  DBR_ERR_PLUGIN,
+  DBR_ERR_MAXERROR,
```

Comment:
I think that last comma is on too many?

Reply:
Yes it shouldn't be there

Reply:
I just updated the code, I added also a better error message